7.2/make_european.py

```diff
@@ -7,6 +7,19 @@
 import re
 from get_input import get_input
 
+def is_valid(user_input):
+    """Take input in the form of a string. Verify the string matches
+    the format:
+    mm/dd/yyyy:username@host
+    Return True for valid, False for invalid.
+    """
+    try:
```

Comment:
You don't need to throw here - the times when you'll need to throw are very few and far between. Rewrite without throwing.